server/src/resolvers/getVariantsResolver/utils/resolveChromosome.ts

```diff
@@ -0,0 +1,14 @@
+const resolveChromosome = (position: string) => {
+  const chromosome = position.split(':')[0];
+  const coordinates = position.split(':')[1];
+  const end = coordinates.split('-')[0];
+  const start = coordinates.split('-')[1];
```

Comment:
If it is actually this simple (that you can just split on these tokens assuming exactly one of them is in `position`), then you can also use a destructuring assignment to not split the same string twice:
```suggestion
  const [chromosome, coordinates] = position.split(':');
  const [end, start] = coordinates.split('-');
```